Util/FileUploader.php

```diff
@@ -153,6 +153,8 @@ private function extractSize($value)
                 case 'G':
                     $size = intval($sizeRegex[1])*10000;
                     break;
```

Comment:
PHP only accepts these suffix : M G K
So the defaults case means we have a value but with an incorrect suffix. I think we should raise a warning, and maybe use the value as octet for safety reason ?
And I forgot a case : we can have a directive without suffix ( = value as octet), so we should have a 
`elseif (isset($sizeRegex[1])) {
 $size = intval($sizeRegex[1];
}
`

Can you make the correction ? thanks :)

Reply:
done